txn_writer.go

```diff
@@ -316,13 +321,13 @@ func (sw *SafeWriter) Write(root string, sm gps.SourceManager, noExamples bool)
 
 		// 0666 is before umask; mirrors behavior of os.Create (used by
 		// writeFile())
-		if err = ioutil.WriteFile(filepath.Join(td, ManifestName), append([]byte(initOutput), tb...), 0666); err != nil {
+		if err := writeFile(filepath.Join(td, ManifestName), sw.Manifest, initOutput); err != nil {
```

Comment:
Oh, there's even a nice comment above explaining that `os.Create()` (which `writeFile` uses) also uses `0666`, I just suck at reading